webclient/package.json

```diff
@@ -37,8 +37,7 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject",
-    "postinstall": "./copy_shared_files.sh"
```

Comment:
And let's keep it as part of the post-install.

Reply:
Ok, I can put this back